ross/rotor_assembly.py

```diff
@@ -3452,3 +3487,65 @@ def MAC_modes(U, V, n=None, plot=True):
     cbar.set_label("MAC")
 
     return macs
+
+
+def rotor_example_6dof():
+    """This function returns an instance of a simple rotor with
+    two shaft elements, one disk and two simple bearings.
+    The purpose of this is to make available a simple model
+    so that doctest can be written using this.
+
+    Parameters
+    ----------
+
+    Returns
+    -------
+    An instance of a 6DoFs rotor object.
+
+    Examples
+    --------
+    >>> rotor = rs.rotor_assembly.rotor_example_6dof()
+    >>> modal = rotor.run_modal(speed=0)
+    >>> np.round(modal.wd[:4])
+    array([ X  X  X  X ])
```

Comment:
I think u meant:

```
>>> import ross as rs
>>> rotor = rs.rotor_example_6dof()
>>> modal = rotor.run_modal(speed=0)
>>> np.round(modal.wd[:4])
array([ X  X  X  X ]) # here, you have to collect the right output
```

Reply:
Correct, we will be creating the test functions and proper example values until next Monday, if it all goes according to plan. By then we will have placed proper numerical values for this test.